secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -118,6 +123,10 @@ private static String randomSecretId() {
     return "java-" + System.currentTimeMillis();
   }
 
+  private static String randomSecretId(String secretPrefix) {
+    return secretPrefix + "-" + System.currentTimeMillis();
```

Comment:
I think best practice is to avoid using `System.currentTimeMillis()` for random number generation - you could consider using `java.util.Random` instead.

Reply:
updated existing method